### PR TITLE
image-mirroring: Drop `driver-toolkit` from OKD 4.{13,14}

### PR DESCRIPTION
**Pull request by LorbusChris**

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_14 [hunk 1]
```diff
@@ -128,7 +128,6 @@ registry.ci.openshift.org/origin/4.14:descheduler quay.io/openshift/origin-desch
 registry.ci.openshift.org/origin/4.14:docker-builder quay.io/openshift/origin-docker-builder:4.14 quay.io/openshift/origin-docker-builder:4.14.0
 registry.ci.openshift.org/origin/4.14:docker-registry quay.io/openshift/origin-docker-registry:4.14 quay.io/openshift/origin-docker-registry:4.14.0
 registry.ci.openshift.org/origin/4.14:dpu-network-operator quay.io/openshift/origin-dpu-network-operator:4.14 quay.io/openshift/origin-dpu-network-operator:4.14.0
-registry.ci.openshift.org/origin/4.14:driver-toolkit quay.io/openshift/origin-driver-toolkit:4.14 quay.io/openshift/origin-driver-toolkit:4.14.0
```
Comment (ybettan) [inline core-services/image-mirroring/openshift/mapping_origin_4_14:131]: Why do we care about older versions? Can't we just keep machine-os-content for older version of OCP and only remove it in newer versions?

Comment (LorbusChris) [reply core-services/image-mirroring/openshift/mapping_origin_4_14:131]: machine-os-content is not touched here. This change removes an OCP CI image that has been leaking into OKD releases (driver-toolkit).

**Pull request merged**